src/features/embalm/stepContent/components/ProgressTrackerStage.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex, Spinner, Text } from '@chakra-ui/react';
+import { StatusIndicator } from './StatusIndicator';
+
+export enum StageStatus {
+  NOT_STARTED = 'NOT_STARTED',
+  IN_PROGRESS = 'IN_PROGRESS',
+  COMPLETE = 'COMPLETE',
+}
+
+export interface ProgressTrackerStageProps {
+  stageStatus?: StageStatus;
+  index?: number;
+  children: React.ReactNode;
+}
+
+export function ProgressTrackerStage({
+  stageStatus = StageStatus.NOT_STARTED,
+  index = 0,
+  children,
+}: ProgressTrackerStageProps) {
+  return (
+    <Flex
+      alignItems="center"
+      justifyContent="space-between"
+    >
+      <Flex>
+        <StatusIndicator
+          status={stageStatus}
+          index={index}
+        />
+        <Text
+          ml={3}
+          fontSize="md"
+          color={stageStatus === StageStatus.NOT_STARTED ? 'brand.500' : 'brand.950'}
+        >
+          {children}
```

Comment:
I don't think this is a problem here, but I'm wondering if having children inside `<Text>` which renders as a `<p>` by default. In this case there is only raw text as children, but we could pass in something that would cause invalid HTML (if anyone actually cared about a thing like that). Maybe a different pattern or check?

Reply:
Yeah, we should add some checks to make sure a dev is adding a string here.